pkg/models/load_balancer.go

```diff
@@ -138,22 +145,92 @@ type CreateLBMonitor struct {
 }
 
 type CreateLBMonitorReq struct {
-	ID                 int    `json:"-" tf:"id,computed"`
-	Name               string `json:"name"`
-	Description        string `json:"description"`
-	MonitorType        string `json:"monitorType" tf:"monitor_type"`
-	MonitorTimeout     int    `json:"monitorTimeout" tf:"monitor_timeout"`
-	MonitorInterval    int    `json:"monitorInterval" tf:"monitor_interval"`
-	SendVersion        string `json:"sendVersion" tf:"send_version"`
-	SendType           string `json:"sendType" tf:"send_type"`
-	ReceiveCode        string `json:"receiveCode" tf:"receive_code"`
-	MonitorDestination string `json:"monitorDestination" tf:"monitor_destination"`
-	MonitorReverse     bool   `json:"monitorReverse" tf:"monitor_reverse"`
-	MonitorTransparent bool   `json:"monitorTransparent" tf:"monitor_transparent"`
-	MonitorAdaptive    bool   `json:"monitorAdaptive" tf:"monitor_adaptive"`
-	FallCount          int    `json:"fallCount" tf:"fall_count"`
-	RiseCount          int    `json:"riseCount" tf:"rise_count"`
-	AliasPort          int    `json:"aliasPort" tf:"alias_port"`
+	ID                  int                         `json:"-" tf:"id,computed"`
+	LbID                int                         `json:"-" tf:"lb_id"`
```

Comment:
ditto

Reply:
LbID is required for creating the monitor from loadbalancer 
here LbID is the  loadbalancer ID 
ID is the monitorID